waffle-waffle/LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright 2019 Marek Kirejczyk
```

Comment:
Hmmm... We could change to Ethworks... or list key contribiutors

Reply:
Will change to `Copyright 2020 Ethworks sp z o.o.`